app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -457,8 +457,9 @@ abstract class BaseBrowserFragment : Fragment(), UserInteractionHandler,
 
             downloadFeature.onDownloadStopped = { downloadState, _, downloadJobStatus ->
                 // If the download is just paused, don't show any in-app notification
-                if (downloadJobStatus == DownloadState.Status.COMPLETED ||
-                    downloadJobStatus == DownloadState.Status.FAILED
+                if ((downloadJobStatus == DownloadState.Status.COMPLETED ||
+                    downloadJobStatus == DownloadState.Status.FAILED) &&
```

Comment:
As we are not performing the download with download manager are we going to get this call?

Reply:
Tested now and seems like we will not. 
Think it's a leftover from an initial version of the AC work.
Thank you!

Reply:
👍🏽